v21.1/primary-key.md

```diff
@@ -4,13 +4,11 @@ summary: A primary key constraint specifies columns that can be used to uniquely
 toc: true
 ---
 
-The `PRIMARY KEY` [constraint](constraints.html) specifies that the constrained columns' values must uniquely identify each row.
+The `PRIMARY KEY` [constraint](constraints.html) specifies that the constrained columns' values must uniquely identify each row. Tables can only have one primary key, but they can have multiple [unique constraints](unique.html).
```

Comment:
```suggestion
The `PRIMARY KEY` [constraint](constraints.html) specifies that the constrained columns' values must uniquely identify each row.  A table can only have one primary key, but it can have multiple [unique constraints](unique.html).
```

Reply:
I think singular `table` to plural `tables` is easier to parse and understand, but up to you.